common/frame_session.go

```diff
@@ -572,7 +571,7 @@ func (fs *FrameSession) onExecutionContextsCleared() {
 
 func (fs *FrameSession) onFrameAttached(frameID cdp.FrameID, parentFrameID cdp.FrameID) {
 	fs.logger.Debugf("NewFrameSession:onFrameAttached",
-		"sid:%v tid:%v fid:%v pfid:%v",
+		"sid:%v tid:%v fid:%v ptid:%v",
```

Comment:
Yeah, still not sure about the `t` :smile:

`pfid` as short for `parentFrameID` makes more sense to me.

Reply:
`t` means `targetID`. I must have confused it while navigating in thousands of lines of code 😅